subprojects/gradle/slack-test-fixtures/src/main/kotlin/com/avito/slack/model/StubFoundMessage.kt

```diff
@@ -6,12 +6,12 @@ fun FoundMessage.Companion.createStubInstance(
     timestamp: String = "",
     emoji: String? = null,
     author: String? = null,
-    channel: SlackChannel = SlackChannel("#")
+    channelId: SlackChannelId = SlackChannelId("C01D88JT6CX")
```

Comment:
Duplicate defaults from subprojects/gradle/lint-report/src/test/kotlin/com/avito/android/lint/ValidGradleTaskNameTest.kt
Let's reuse the value